tests/unittest.py

```diff
@@ -203,6 +204,7 @@ def create_peer(  # type: ignore[no-untyped-def]
         enable_event_queue: bool | None = None,
         enable_ipv6: bool = False,
         disable_ipv4: bool = False,
+        mock_peers_whitelist: URLPeersWhitelist | FilePeersWhitelist | None = None,
```

Comment:
It doesn't have to be mocked. You can accept any `PeersWhitelist | None` or simply `str` and use the factory.